akka-remote/src/test/scala/akka/remote/RemoteRouterSpec.scala

```diff
@@ -20,8 +20,14 @@ object RemoteRouterSpec {
   }
 }
 
-class RemoteRouterSpec extends AkkaSpec("""
+class RemoteRouterSpec extends AbstractRemoteRouterSpec(false)
+class RemoteRouterJavaSerializationSpec extends AbstractRemoteRouterSpec(true)
```

Comment:
I think we can skip testing those with JavaSerialization in master. Not sure that increasing the testing time for this is worth it.